connectors-home/modules/ROOT/pages/introduction/intro-config-use-acb.adoc

```diff
@@ -0,0 +1,220 @@
+. Populate your file with the connection properties for your connector, for example, your Salesforce, Gmail, and Slack configurations:
++
+[source,YAML]
+--
+salesforce:
+ username: "yourUserName"
+ password: "yourPassword"
+ token: "yourToken"
+
+email:
+ host: "smtp.gmail.com"
+ username: "yourGmailEmail"
+ password: "yourEmailAppPassword"
+
+slack:
+ token: "yourSlackToken"
+--
+. Navigate to your project XML, add a new line before the `</mule>` tag, type `configuration` and select *mule:configurationProperties*:
++
+image::anypoint-code-builder::select-mule-config-properties-contacts-sync.png["List of components with an arrow pointing to the mule:configurationProperties component"]
+. Scroll through the attributes and set the value of `file` to `config.yaml`:
++
+[source,XML]
+--
+<configuration-properties file="config.yaml" doc:name="Configuration properties" />
+--
+
+=== Salesforce Connector Configuration Example
+
+Use the values in your `config.yaml` file to connect Salesforce Connector to your Salesforce environment:
+
+. In your project XML file, create a new line after the `<configuration-properties/>` element, then type `salesforce`, and select *salesforce:sfdc-config*.
+. Set the *name* field to `Salesforce-Config`.
+. Create a new line, type `basic` and select *Salesforce:Basic*.
+. Configure the `username` and `password` properties to use your configuration file values:
++
+[source,XML]
+--
+<salesforce:basic-connection username="${salesforce.username}" password="${salesforce.password}" securityToken="${salesforce.token}" />
+--
++
+[NOTE]
+--
+When you move your cursor over the `${salesforce.username}` and `${salesforce.password}` elements, Anypoint Code Builder shows the values from the configuration file.
+--
+. Click *Test Connection* for the `<salesforce:sfdc-config/>` element to verify your connection to Salesforce:
++
+image::anypoint-code-builder::sync-api-sfdc-test-connection.png["Arrow pointing to the Test Connection option in the connection XML"]
++
+Anypoint Code Builder returns a *Connection is valid* message.
++
+If you get an error, verify the error message to troubleshoot further.
+
+
+[[configure-trigger]]
+== Configuring a Trigger for the Flow
+// tag::acb-configure-source[]
+A trigger (source) starts the flow when specified conditions are met. If a connector has its own associated sources, you can use a connector-specific trigger to initiate the flow. Each connector trigger has a unique configuration.
+
+=== Using HTTP Listener as a Trigger
+
+. Click the *Listener* component on the canvas.
+. (Optional) Click the edit icon to change the name of the *Listener* component. 
+. (Optional) If you created a configuration file for the connector, select it. 
+. In the *General* tab, in the *Path* field, enter the relative path for the path set in the HTTP configuration element. See xref:http/http-listener-ref.adoc#configure-paths[Configure Paths]. 
+. Select the *Advanced* tab to configure the following attributes:
```

Comment:
In this case, since the config UI is very incomplete, some of it has to be done in the XML. I'm staying consistent with ACB terminology, which calls these attributes.

Reply:
Then change the column in the field description table to Attribute instead of Field.